config.js

```diff
@@ -19,6 +19,10 @@
     "featureManagerIsSSLEnabled": false,
     "featureManagerIsWAFEnabled": true,
     "homePageCards": ["ApplyDefaultSettingsCard", "SSLCard", "PurgeCacheCard"],
+    "moreSettingsCards": {
+        "container.moresettings.speed": ["CacheLevelCard", "AlwaysOnlineCard", "ImageOptimizationCard"],
+        "container.moresettings.security": ["SecurityLevelCard"]
+    },
```

Comment:
Good point, it doesn't really matter, as long as something is there its fine.  Don't worry about changing this.